src/boost-api/Endpoint_Handlers.cpp

```diff
@@ -16,10 +16,10 @@ namespace api {
     }
 
     boost::json::value get_json_from_request_body(const request<string_body> &req, response<string_body> &res) {
```

Comment:
Response bliver vel ikke brugt til noget i denne metode?

```suggestion
    boost::json::value get_json_from_request_body(const request<string_body> &req) {
```

Reply:
Expand this (tracer voice)